The-one-and-only-week1/readme.md

```diff
@@ -1,5 +1,30 @@
 # Order form exercise
```

Comment:
It's confusing to have two H1 titles next to each other. Maybe move "Order form exercise" below the html best practices section?